tool/LICENSE

```diff
@@ -173,3 +173,30 @@
       defend, and hold each Contributor harmless for any liability
       incurred by, or claims asserted against, such Contributor by reason
       of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
```

Comment:
Why was this added?

Reply:
I think this was auto generated possibly due to the upgrade of monorepo